CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog Release Management
+
+## Table of Contents
+
+- **[r1.1](#r11)**
+
+# r1.1
+
+## Release Notes
+
+**This is the first pre-release of the Release Management documents with version 0.1.0.** 
+
+Content is based on the work done by the [Release Management Working Group](https://lf-camaraproject.atlassian.net/l/cp/scZC1AYy) and the results are located in the [Release Management repository](https://github.com/camaraproject/ReleaseManagement).
+
+**It provides the initial version of de guidelines describing how to releasing an API version, as well as the API-Readiness-Checklist  template in which to record the availability of the artefacts required for the API release.**
+
+It contains the Release Management approved documents in the **documentation** folder:
+   - **[API_Release_Guidelines.md](https://github.com/camaraproject/ReleaseManagement/blob/r1.1/documentation/API_Release_Guidelines.md)** - An extract of the API Release Management process form the CAMARA wiki.
+   - **[API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/r1.1/documentation/API-Readiness-Checklist.md)** - The checklist to ensure release readiness of the API according to its release type (alpha, release-candidate, public).
+
+### Added
+* Add API-Readiness-Checklist.md to RM project by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/29
+* Add API Release Guidelines to GitHub - extract of the API Release Pro… by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/31
+* Update public release name by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/35
+* Update readme.md and further links in rm project material by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/42
```

Comment:
Based on the auto-generated changelist from GitHub:

```suggestion
* Add API-Readiness-Checklist.md to RM project by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/29
* Add API Release Guidelines to GitHub - extract of the API Release Pro… by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/31
* Added changelog template and example by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/36 and @hdamker in https://github.com/camaraproject/ReleaseManagement/pull/38
* Update public release name by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/35
```
#42 was merged end of July.